dc_utils/templates/dc_forms/field_checkbox.html

```diff
@@ -2,20 +2,21 @@
 <div class="ds-field">
     {% if field|is_multiple_checkbox %}
         <fieldset>
-            <legend>{{ field.label }}</legend>
+            <legend>{{ field }}</legend>
             <div class="ds-stack-smallest">
                 {% for radio in field %}
-                    <label class="ds-field-checkbox" for="{{ radio.id_for_label }}">
+                    <label class="ds-field-checkbox" for="{% if VERSION >= 4 %}{{ radio.id_for_label }}{% else %}{{ radio }}{% endif %}">
                         {{ radio.tag }}
-                        <span>{{ radio.choice_label }}</span>
+                        <span>{% if VERSION >= 4 %}{{ radio.choice_label }}{% else %}{{ radio }}{% endif %}</span>
                     </label>
                 {% endfor %}
             </div>
         </fieldset>
     {% else %}
         <label class="ds-field-checkbox">
             {{ field }}
-            <span>{{ field.label }}</span>
+            <span>{% if VERSION >= 4 %}{{ field.label }}{% else %}{{ field.label }}{% endif %}</span>
```

Comment:
Just adding the version to the settings won't expose it in the template layer (this is to protect things like the database password from getting to templates), so you'll have to find another way to get this var to the templates. 

I suggest adding this to [the context processor](https://github.com/DemocracyClub/dc_django_utils/blob/main/dc_utils/context_processors.py).

This also means that your tests are running because `if VERSION >= 4` is _always_ false at the moment.